### PR TITLE
Dynamic surveillance positives

### PR DESCRIPTION
**Pull request by ereng**
@briankip please review

Configure test types of the diseases
NB: works only for alphanumeric ranges

Side bar > Test Types > Measures > Ranges > interpretations

ranges that mean positive put in the interpretation field '+' be sure not to add any white space before or after it

look out for Test Types of interest
- Malaria Test
- Bs for mps
- Widal Test
- salmonella antigen test
- Stool for C/S

### CODE REVIEW

File: app/models/Test.php [hunk 1]
```diff
@@ -568,4 +577,48 @@ public static function getSurveillanceData($from, $to)
...
+
+		foreach ($measureIds as $measureId) {
+
+			$measure = Measure::find($measureId);
+
+			$measureRanges = $measure->measureRanges;
+
+			foreach ($measureRanges as $measureRange) {
+
+				if ($measureRange->interpretation == '+') {
```
Comment (briankip) [inline app/models/Test.php]: Isn't '+' still hardcoding, ++ is still positive in some cases? So is +++ and ++++ e.t.c

File: app/models/Test.php [hunk 2]
```diff
@@ -568,4 +577,48 @@ public static function getSurveillanceData($from, $to)
...
+		}
+
+		return $positiveRanges;
+	}
+
+	/**
+	 * @param  Test Type IDs $testTypeIds array()
+	 * @return Measure IDs $measureIds array()
+	 */
+	public static function getMeasureIdsByTestTypeIds($testTypeIds)
```
Comment (briankip) [inline app/models/Test.php]: Can testType->measure->lists('id') work?

Comment (ereng) [inline app/models/Test.php]: yep.

File: app/models/Test.php (hunk 1 above)
Comment (ereng) [inline app/models/Test.php]: true... will put as a constant in the right place for now

**Comment by ereng** [comment]
@kitsao I think this would work in BDH as well, please look at it.

**Comment by kitsao** [comment]
I don't get it. I have a headache right now, I'll comprehend it later.

**Pull request merged**